web-ui/src/components/feedback_submit_form/FeedbackSubmitForm.jsx

```diff
@@ -82,63 +167,84 @@ const FeedbackSubmitForm = ({requesteeName, requestId}) => {
       let questionsList = res.questions ? res.questions : [];
       return questionsList;
     }
+
+    async function getAnswers(questionsList) {
+      if (!questionsList) {
+        return;
+      }
+      const res = await getAllAnswersFromRequestAndQuestionId(requestId, questionsList, csrf)
```

Comment:
I don't think the await keyword is needed here as no operations are performed on the response. You can just return the promise directly.   The same is true for lines 111, 102, and 93.

Reply:
Fixed